apps/playground-api/__tests__/PlaygroundBlock.test.ts

```diff
@@ -0,0 +1,68 @@
+import { PlaygroundApiTesting } from '../testing/PlaygroundApiTesting'
+import waitForExpect from 'wait-for-expect'
+import { PlaygroundProbeIndicator } from '../src/PlaygroundIndicator'
+import { PlaygroundBlock } from '../src/PlaygroundBlock'
+import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { StartOptions, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { mining } from '@defichain/jellyfish-api-core'
+
+class PlaygroundApiMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
+  protected getCmd (opts: StartOptions): string[] {
+    return [
+      ...super.getCmd(opts),
+      `-masternode_operator=${RegTestFoundationKeys[1].operator.address}`,
```

Comment:
may be good to add a comment that why we add 7 MN operators here (helps newbie like me to understand)